print_docs.py

```diff
@@ -646,6 +646,20 @@ def write_site_map(partition):
     out.write(site_root + filename + '.html\n')
   out.close()
 
+def redirect_body(decl_name, decl_loc):
+  url = filename_core(site_root, decl_loc, 'html')
+  return f"""
+<!DOCTYPE html>
+<Content-Type="text/html">
```

Comment:
```suggestion
<meta http-equiv="content-type" content="text/html">
```
Are these two lines necessary at all?

Reply:
Probably not, I'll test. This was part of my failed attempt to make it work with the Python local server.